### PR TITLE
feat(api): add score_qualite_minimum filter

### PR DESCRIPTION
**Pull request by hlecuyer**

### CODE REVIEW

File: api/tests/e2e/api/test_inclusion_data.py [hunk 1]
```diff
@@ -663,6 +663,33 @@ def test_list_services_by_types(api_client, url):
...
+        service_1.id,
+        service_2.id,
+    }
+
+    response = api_client.get(
+        url,
+        params={"score_qualite_minimum": 0.8},
+    )
+    assert_paginated_response_data(response.json(), total=0)
+
```
Comment (vmttn) [inline api/tests/e2e/api/test_inclusion_data.py:691]:
```suggestion
```

je crois que cette partie n'ajoute rien de plus, en l'état je dirais de retirer

File: api/tests/e2e/api/test_inclusion_data.py [hunk 2]
```diff
@@ -663,6 +663,33 @@ def test_list_services_by_types(api_client, url):
...
+@pytest.mark.with_token
+@pytest.mark.parametrize("url", ["/api/v0/services", "/api/v0/search/services"])
+def test_list_services_by_score_qualite(api_client, url):
+    service_1 = factories.ServiceFactory(score_qualite=0.5)
+    service_2 = factories.ServiceFactory(score_qualite=0.7)
+    factories.ServiceFactory(score_qualite=0.2)
+
+    response = api_client.get(
+        url,
+        params={"score_qualite_minimum": 0.5},
```
Comment (vmttn) [inline api/tests/e2e/api/test_inclusion_data.py:675]: :+1: nickel d'avoir mis ce edge case

peut-être :

```suggestion
        params={"score_qualite_minimum": service_1.score_qualite},
```

pour bien voir ce que la valeur n'est pas choisie au hasard ?

**Review by vmttn — CHANGES_REQUESTED** [review]
almost perfect :)

**Comment by hlecuyer** [comment]
@vmttn done :)
